[IMP] inventory: delivery orders and incoming shipments restructure

samueljlieber opened the pull request:
Tasks: [#3205819](https://www.odoo.com/web#id=3205819&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form), [#3208499](https://www.odoo.com/web#id=3208499&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form), [#3208472](https://www.odoo.com/web#id=3208472&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form), [#3210922](https://www.odoo.com/web#id=3210922&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form), [#3189292](https://www.odoo.com/web#id=3189292&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form), [#3189335](https://www.odoo.com/web#id=3189335&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

Targets: 14.0, 15.0, & 16.0

This PR restructures the **Delivery Orders** and **Incoming Shipments** folders by consolidating both folders into a single **Incoming Shipments & Delivery Orders** folder and merges the following docs:

- `delivery/inventory_flow.rst` ➡️ `shipments_deliveries/inventory_flow.rst`
- `delivery/one_step.rst` & `incoming/one_step.rst` ➡️ `shipments_deliveries/receipts_delivery_one_step.rst`
- `delivery/two_steps.rst` & `incoming/two_step.rst` ➡️ `shipments_deliveries/receipts_delivery_two_steps.rst`
- `delivery/three_steps.rst` ➡️ `shipments_deliveries/delivery_three_steps.rst`
- `incoming/three_steps.rst` ➡️ `shipments_deliveries/receipts_three_steps.rst`

As well as moved two files from **Delivery Orders** to a more appropriate location for the file content **Shipping Operations**:

- `delivery/label_type.rst` ➡️ `shipping/operation/label_type.rst`
- `delivery/dropshipping.rst` ➡️ `shipping/operation/dropshipping.rst`

Here are images showing the [old structure](https://drive.google.com/file/d/1Z609RlbDsKYJjrQ7czUORv0WhsdMu4Tt/view?usp=sharing) and the [new structure](https://drive.google.com/file/d/1BTBulw4Ch6sojqWWiuypfz9BAA2aorq5/view?usp=sharing) created by this PR.

Redirects were added for all file moves and name changes.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/3762).

samueljlieber commented:
Hi @larm-odoo 👋 
Great job on writing up the RST and coordinating the merge of these docs. I pushed the PR up as a draft so you can review the content within it's new structure for accuracy before we send it over for a content review (set up a time with me and I can show you how to pull this PR down to your computer so you can review 🙂).

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+====================
+
+In the :guilabel:`Sales` application, create a quotation for an international customer with a
+storable product. Click on :guilabel:`Create`, and select an international customer for the quote.
```
Delete "on" and the "," in: Click on :guilabel:`Create`,

Instead: Click :guilabel:`Create`

larm-odoo reviewed (changes requested):
Hi @sali - here are just a few changes. I can't believe I didn't see the until I saw them in this format!

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+**Shipping**: Ship products directly from stock.  No intermediate steps between stock and shipping
+occur, such as a transfer to a packing location.
+
+- Can only be used if not using :abbr:`FIFO (First In, First Out), :abbr:`LIFO (Last In, First
```
Looks like I am missing a "`"  here:
 :abbr:`FIFO (First In, First Out)`,
 
 The FIFO is not showing up on the html doc, and also, i don't see the comma between the two listed and there should be commas between all of them.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+different carriers or shipping docks before being shipped.
+
+- Minimum requirement for tracking products by lot or serial numbers with a :abbr:`FIFO (First In,
+  First Out)`, :abbr:`LIFO (Last In, First Out)` and :abbr:`FEFO (First Expired, First Out)` removal
```
should have a comma after LIFO, and instead of "and" it should be "or"

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+  strategy.
+- Recommended for larger warehouses with high stock levels, or when stocking large items (such as
+  mattresses, large furniture, heavy machinery, etc.).
+- Products received will not be available for manufacturing, shipping, etc, until they are
```
add a period after etc

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+.. seealso::
+   - :doc:`receipts_delivery_one_step`
+
+Two steps flow
```
Two step flow

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+.. seealso::
+   - :doc:`receipts_delivery_two_steps`
+
+Three steps flow
```
Three step flow

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+**Pick + pack + ship**: Pick products according to the removal strategy, then create packages in a
+dedicated packing area, and finally, bring them to an output location for shipping.
+
+- Can be used when tracking products by lot or serial numbers when using :abbr:`FIFO (First In,
```
when using a (add the "a")

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+dedicated packing area, and finally, bring them to an output location for shipping.
+
+- Can be used when tracking products by lot or serial numbers when using :abbr:`FIFO (First In,
+  First Out)`, :abbr:`LIFO (Last In, First Out)` and :abbr:`FEFO (First Expired, First Out)` removal
```
"or" instead of "and"

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/inventory_flow.rst:
```diff
@@ -0,0 +1,76 @@
+- Recommended for very large warehouses with very high stock levels.
+- Required for any warehouse needing to perform quality control inspections before receiving items
+  into stock.
+
```
Add:
-Products received will not be available for manufacturing, shipping, etc., until they are transferred into stock.

larm-odoo reviewed (changes requested):
Hi @sali, some more changes I noticed.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_one_step.rst:
```diff
@@ -0,0 +1,111 @@
+from the stock to the customer.
+
+Both incoming and outgoing shipments do not need to be configured to have the same steps. For
+example products can be received in one step, but shipped in three steps. In the following example,
```
example, (add comma)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_one_step.rst:
```diff
@@ -0,0 +1,111 @@
+to one step. Go to :menuselection:`Inventory --> Configuration --> Warehouses`, click on the
+warehouse to edit, then, click :guilabel:`Edit`. Select :guilabel:`Receive goods directly (1 step)`
+for :guilabel:`Incoming Shipments` and :guilabel:`Deliver goods directly (1 step)` for
+:guilabel:`Outgoing Shipments`, then save the changes
```
:guilabel:`Save`

larm-odoo reviewed (changes requested):
Just a couple of small items, @sali - one is an image that needs to be swapped.

larm-odoo reviewed (changes requested):
Just a couple of small items, @sali - one is an image that needs to be swapped.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+In the two steps delivery  process, products which are part of a delivery order are picked from the
+warehouse according to the removal strategy, and brought to an output location before being shipped.
+One situation where this would be useful is when using either a :abbr:`FIFO (First In, First Out)`,
+:abbr:`LIFO (Last In, First Out)` and :abbr:`FEFO (First Expired, First Out)`  removal strategy,
```
"or" instead of "and"

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+warehouse according to the removal strategy, and brought to an output location before being shipped.
+One situation where this would be useful is when using either a :abbr:`FIFO (First In, First Out)`,
+:abbr:`LIFO (Last In, First Out)` and :abbr:`FEFO (First Expired, First Out)`  removal strategy,
+where the products that are being picked need to be selected based on their lot number, receipt
```
delete "lot number,"

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+One situation where this would be useful is when using either a :abbr:`FIFO (First In, First Out)`,
+:abbr:`LIFO (Last In, First Out)` and :abbr:`FEFO (First Expired, First Out)`  removal strategy,
+where the products that are being picked need to be selected based on their lot number, receipt
+date, or expiration date.
```
remove the comma, only two items in the list now.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+   :alt: After confirming the sales order, the Delivery smart button appears showing two items
+         associated with it.
+
+Process a Picking
```
picking (lowercase)

larm-odoo replied:
picking (lower case)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+   :alt: Validate the receipt by clicking Validate, then the product will be transferred to the
+         WH/Input location.
+
+Process an Internal Transfer
```
internal transfer (lower case)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+:menuselection:`Inventory --> Configuration --> Warehouses`, edit the warehouse, then, select
+:guilabel:`Receive goods in input and then stock (2 steps)` for :guilabel:`Incoming Shipments` and
+:guilabel:`Send goods in output and then deliver (2 steps)` for :guilabel:`Outgoing Shipments`, then
+save the changes.
```
:guilabel:`Save`

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+
+Once the product is in the :guilabel:`Input Location`, the internal transfer is ready to move the
+product to :guilabel:`Stock`. In the inventory overview dashboard, click the :guilabel:`1 To
+Process` smart button in the :guilabel:`Internal transfers` Kanban card.
```
Transfers (capital T)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+   :alt: One Internal Transfer ready to process in the Inventory Overview Kanban view.
+
+Click on the transfer associated with the purchase order, then click :guilabel:`Validate` to
+complete the receipt and move the product to the :guilabel:`Stock`. Once the transfer is validated,
```
the product to Stock (delete "the")

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,121 @@
+   :alt: One Receipt ready to process in the Inventory Overview Kanban view.
+
+Click on the receipt associated with the purchase order, then click :guilabel:`Validate` to complete
+the receipt and move the product to the :guilabel:`Quality Location`.
```
:guilabel:`Input Location` (not quality)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,121 @@
+
+Click on the transfer associated with the purchase order, then click :guilabel:`Validate` to
+complete the transfer and move the product to the :guilabel:`Quality Control` location. Once the
+transfer is validated, the product is ready for the quality inspection.
```
ready for the quality inspection, but is not available for manufacturing or delivery orders.

larm-odoo reviewed (changes requested):
Just a couple of small items, @sali - one is an image that needs to be swapped.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst:
```diff
@@ -0,0 +1,136 @@
+Next, the warehouse needs to be configured for three step deliveries. Go to
+:menuselection:`Inventory --> Configuration --> Warehouses`, edit the warehouse, then, select
+:guilabel:`Pack goods, send goods in output and then deliver (3 steps)` for
+:guilabel:`Outgoing Shipments`, then save the changes.
```
:guilabel:`Save`

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst:
```diff
@@ -0,0 +1,136 @@
+   :alt: After confirming the sales order, the Delivery smart button appears showing three items
+         associated with it.
+
+Process a Picking
```
picking (lower case)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst:
```diff
@@ -0,0 +1,136 @@
+   :align: center
+   :alt: Validate the picking by clicking Validate.
+
+Process a Packing
```
packing (lower case)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst:
```diff
@@ -0,0 +1,136 @@
+moves to the :guilabel:`WH/Output` location. Then, the status of the document will change to
+:guilabel:`Done`.
+
+Process a Delivery
```
delivery (lower case)

larm-odoo reviewed (changes requested):
That's it! This is the last part of this whole huge PR  I reviewed. I am hoping making the changes is easy, and I hope each time I clicked 'submit review" it didn't re-submit all of them? I know I clicked a couple times in two of them, not knowing if I did that or not. So you'll have to tell me if I messed this up or not.

samueljlieber commented:
Hi @larm-odoo! Thank you for your improvements 🙂 I implemented all of them in https://github.com/odoo/documentation/pull/3762/commits/e0fa2d16c1ef6d0ccef92c67fa449561fe37424d. Please review the changes, and approve if you are ready to open this PR up for content review!

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_one_step.rst:
```diff
@@ -0,0 +1,111 @@
+   :align: center
+   :alt: Validate the delivery order.
+
+Once the picking order is validated, the product leaves the :guilabel:`WH/Stock location` and moves
```
change picking order to delivery order

larm-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+in input, then quality and then stock (3 steps)` for :guilabel:`Incoming Shipments`, then save the
+changes.
+
+.. image:: reciepts_three_steps/three-step-warehouse-config.png
```
please change image to three-step-receipt-setting.png.

larm-odoo reviewed (changes requested):
OK, only TWO changes then we are GOOD TO SUBMIT!

larm-odoo reviewed (approved):
Let's DO THIS THING!

StraubCreative commented:
@brse-odoo can you give this a peer review please and then tag @hojo-odoo ? TIA 🙏

samueljlieber commented:
Rebased to latest in https://github.com/odoo/documentation/commit/3d2a5e516276e232cb54e30033776c70ae769b4b

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+
+Depending on a company's business processes, multiple steps may be needed before receiving or
+shipping products. In the two step receipt process, products are received in an input area, then
+transferred to stock. An example of using two step receipts is if various storage locations are
```
```suggestion
transferred to stock. Two step receipts works best when various storage locations are
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+shipping products. In the two step receipt process, products are received in an input area, then
+transferred to stock. An example of using two step receipts is if various storage locations are
+being used, such as locked or secured areas, freezers and refrigerators, or various shelves.
+Products can be sorted according to where they are going to be stored, and employees can stock all
```
Separate this into a new paragraph (starting with "Products...")

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+
+In the two steps delivery  process, products which are part of a delivery order are picked from the
+warehouse according to the removal strategy, and brought to an output location before being shipped.
+One situation where this would be useful is when using either a :abbr:`FIFO (First In, First Out)`,
```
Separate this into a new paragraph (starting with "One...")

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+
+Odoo is configured by default to :doc:`receive and deliver goods in one step
+<receipts_delivery_one_step>`, so the settings need to be changed in order to utilize two-step
+receipts and deliveries. Both incoming and outgoing shipments do not need to be set the same steps.
```
```suggestion
receipts and deliveries. Both incoming and outgoing shipments do not need to be set to the same steps.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+
+Next, the warehouse needs to be configured for two step receipts and deliveries. Go to
+:menuselection:`Inventory --> Configuration --> Warehouses`, edit the warehouse, then, select
+:guilabel:`Receive goods in input and then stock (2 steps)` for :guilabel:`Incoming Shipments` and
```
```suggestion
:guilabel:`Receive goods in input and then stock (2 steps)` for :guilabel:`Incoming Shipments`, and
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+
+.. image:: receipts_delivery_two_steps/two-step-po-receipt.png
+   :align: center
+   :alt: After confirming a purchase order, a :guilabel:`Receipt` smart button will appear.
```
I don't think we need rst in image alt tags?

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+--------------------
+
+In the :guilabel:`Sales` application, create a new quote by clicking :guilabel:`Create`. Select a
+customer, add a storable product, and click :guilabel:`Confirm`.
```
`:guilabel:` for customer and product

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+picking will be :guilabel:`Ready`, since the product must be picked from stock before it can be
+shipped. The status of the delivery order will be :guilabel:`Waiting Another Operation`, since the
+delivery cannot happen until the picking is completed. The status of the delivery order will only
+change to :guilabel:`Ready` when the picking has been marked :guilabel:`Done`.
```
```suggestion
change to :guilabel:`Ready` once the picking has been marked as :guilabel:`Done`.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+   :align: center
+   :alt: The pick order can be seen in the Inventory Kanban view.
+
+Click on the picking to process. If the product is in stock, Odoo will automatically reserve the
```
```suggestion
Click on the :guilabel:`Picking` to process. If the product is in stock, Odoo will automatically reserve the
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_delivery_two_steps.rst:
```diff
@@ -0,0 +1,181 @@
+   :align: center
+   :alt: The delivery order can be seen in the Inventory Kanban view.
+
+Click on the delivery order associated with the sales order, then click on :guilabel:`Validate` to
```
If this indicates UI give DO and SO a `:guilabel:`

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+   :alt: Activate multi-step routes and storage locations in inventory settings.
+
+Next, the warehouse needs to be configured for three step receipts. Go to :menuselection:`Inventory
+--> Configuration --> Warehouses`, edit the warehouse, then, select :guilabel:`Receive goods
```
```suggestion
--> Configuration --> Warehouses`, and click :guilabel:`Edit` to edit the :guilabel:`Warehouse`. Then, select :guilabel:`Receive goods
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+
+Next, the warehouse needs to be configured for three step receipts. Go to :menuselection:`Inventory
+--> Configuration --> Warehouses`, edit the warehouse, then, select :guilabel:`Receive goods
+in input, then quality and then stock (3 steps)` for :guilabel:`Incoming Shipments`, then save the
```
```suggestion
in input, then quality and then stock (3 steps)` for :guilabel:`Incoming Shipments`. Then, :guilabel:`Save` the
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+   :align: center
+   :alt: Set incoming and outgoing shipment options to receive and deliver in three steps.
+
+Activating three step receipts and deliveries will create two new internal locations,
```
```suggestion
Activating three step receipts and deliveries will create two new internal locations:
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+
+Activating three step receipts and deliveries will create two new internal locations,
+:guilabel:`Input` (WH/Input), and :guilabel:`Quality Control` (WH/Quality Control). To rename these
+locations, go to :menuselection:`Configuration --> Locations`, select the location to change, click
```
```suggestion
locations, go to :menuselection:`Configuration --> Locations`, select the :guilabel:`Location` to change, click
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+-----------------------
+
+In the :guilabel:`Purchase` application, create a new quote by clicking :guilabel:`Create`. Select a
+vendor, add a storable product, and click :guilabel:`Confirm Order`.
```
`:guilabels:` on vendor and product

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+
+The status of the first internal transfer to quality will only change to :guilabel:`Ready` when the
+receipt has been marked :guilabel:`Done`. The status for the second internal transfer to stock will
+be marked :guilabel:`Ready` only after the transfer to quality has been marked :guilabel:`Done`.
```
Should "quality" here be anything? `:guilabel:`, *italicized*?

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+   :align: center
+   :alt: One Internal Transfer ready to process in the Inventory Overview Kanban view.
+
+Click on the transfer associated with the purchase order, then click :guilabel:`Validate` to
```
```suggestion
Click on the :guilabel:`Transfer` associated with the purchase order, then click :guilabel:`Validate` to
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+   :align: center
+   :alt: Validate the internal transfer to move the item to the Quality Control location.
+
+Process a transfers to stock
```
"a transfer" or "transfers" ?

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+============================
+
+Once the product is in the :guilabel:`Quality Control` location, the final internal transfer is
+ready to move the product to :guilabel:`Stock`. In the inventory overview dashboard, click the
```
```suggestion
ready to move the product to :guilabel:`Stock`. In the :guilabel:`Inventory` overview dashboard, click the
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/receipts_three_steps.rst:
```diff
@@ -0,0 +1,122 @@
+ready to move the product to :guilabel:`Stock`. In the inventory overview dashboard, click the
+:guilabel:`1 To Process` smart button in the :guilabel:`Internal Transfers` Kanban card.
+
+Click on the final transfer associated with the purchase order, then click :guilabel:`Validate` to
```
```suggestion
Click on the final :guilabel:`Transfer` associated with the purchase order, then click :guilabel:`Validate` to
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+Overview
+========
+
+With Odoo, there are a variety of different types of shipping  labels that can be selected for
```
```suggestion
In Odoo, there are a variety of different types of shipping labels that can be selected for
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+
+With Odoo, there are a variety of different types of shipping  labels that can be selected for
+delivery orders. Depending on the types of shipping packages used, different label sizes may be more
+appropriate, and can be easily configured.
```
```suggestion
appropriate, and can be configured to fit the package.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+=============
+
+In the :guilabel:`Inventory` module, go to :menuselection:`Configuration --> Delivery --> Shipping
+Methods.` Choose a delivery method and then click :guilabel:`Edit`. For the following example,
```
```suggestion
Methods.` Choose a delivery method and click :guilabel:`Edit`. For the following example,
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+
+In the :guilabel:`Inventory` module, go to :menuselection:`Configuration --> Delivery --> Shipping
+Methods.` Choose a delivery method and then click :guilabel:`Edit`. For the following example,
+*FedEx International* will be used, so select a delivery method and then click on :guilabel:`Edit`.
```
This is an awkward sentence - maybe just stop it after *Fedex International*? Totally up to you

brse-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+   :align: center
+   :alt: Select a label type.
+
+When a *sales order* with the corresponding shipping company is confirmed, and the *delivery order*
```
```suggestion
When a *Sales Order* with the corresponding shipping company is confirmed and a *Delivery Order*
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+Create a Sales Order
+====================
+
+In the :guilabel:`Sales` application, create a quotation for an international customer with a
```
```suggestion
In the :guilabel:`Sales` application,
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,73 @@
+====================
+
+In the :guilabel:`Sales` application, create a quotation for an international customer with a
+storable product. Click :guilabel:`Create` and select an international customer for the quote.
```
```suggestion
click :guilabel:`Create` and select an international customer for the quote.
```

brse-odoo reviewed (commented):
@larm-odoo I've finished my review of all the documents in this PR -- great job! I think it's going to go a long way to have these docs be up to snuff and reorganized way more clearly.

I left some comments and suggestions (nothing major) for some small fixes/grammatical stuff to keep things consistent. Feel free to ping me with any questions.

@hojo-odoo -- tagging you per Zac's request. I'm not sure if you want to review this now, or wait until Lara combs through my changes and pushes a new, revised commit. I'll leave that up to you.

Thanks all!

brse-odoo reviewed (commented):
@larm-odoo I've finished my review of all the documents in this PR -- great job! I think it's going to go a long way to have these docs be up to snuff and reorganized way more clearly.

I left some comments and suggestions (nothing major) for some small fixes/grammatical stuff to keep things consistent. Feel free to ping me with any questions.

@hojo-odoo -- tagging you per Zac's request. I'm not sure if you want to review this now, or wait until Lara combs through my changes and pushes a new, revised commit. I'll leave that up to you.

Thanks all!

larm-odoo commented:
Thanks @brse-odoo   - I took all the suggestions and made all the edits. Now, if @hojo-odoo wants to take a look, it's ready!

StraubCreative commented:
Quick note @larm-odoo @brse-odoo : 

• Make sure headings are in [sentence case](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#titles-and-headings).
• Do not capitalize nouns unless they are proper nouns or if they are being reflected from the UI (as in a `guilabel`).

samueljlieber commented:
Hi @larm-odoo 👋 your last commit https://github.com/odoo/documentation/commit/b33e17b84f2d1f2f519ad10d04d7accfd7df15f1 had some issues that I fixed in 5a07b7f . 

Looks like the issue was an image folder for `receipts_three_steps.rst` with the name of `reciepts_three_steps` where the **i** was before the **e** was created and there was already an image folder for that RST file with the correct name - I removed that image folder and the images 👍

I also unresolved a few of @brse-odoo 's suggestions because I noticed they were not implemented or needed corrections while I was fixing the images, listed below. I did not implement the following:

- [guilabel added but needs to be uppercase & missing a period](https://github.com/odoo/documentation/pull/3762#discussion_r1137517765)
- [guilabel added but needs to be uppercase](https://github.com/odoo/documentation/pull/3762#discussion_r1137519740)
- [guilabels added but needs to be uppercase](https://github.com/odoo/documentation/pull/3762#discussion_r1137521946)
- [Line wasn't removed - may be intentional](https://github.com/odoo/documentation/pull/3762#discussion_r1137532866)
- [guilabels added but needs to be uppercase](https://github.com/odoo/documentation/pull/3762#discussion_r1137539451)
- [guilabels added but needs to be uppercase](https://github.com/odoo/documentation/pull/3762#discussion_r1137542623)
- [guilabels added but needs to be uppercase](https://github.com/odoo/documentation/pull/3762#discussion_r1137553804)
- [guilabels added but needs to be uppercase](https://github.com/odoo/documentation/pull/3762#discussion_r1137560194)
- [Missed capitalization on Sales Order](https://github.com/odoo/documentation/pull/3762#discussion_r1137578462)
- [Sentence restructure](https://github.com/odoo/documentation/pull/3762#discussion_r1137579034)
- [Sentence restructure](https://github.com/odoo/documentation/pull/3762#discussion_r1137579253)

And finally, one of the build errors is due to the heading delimiter not being the same length as the heading on line `116` of `receipts_three_steps.rst` 🙂

hojo-odoo commented on content/applications/inventory_and_mrp/inventory/shipping/operation/label_type.rst:
```diff
@@ -0,0 +1,72 @@
+
+In the :guilabel:`Sales` application, click :guilabel:`Create` and select an international customer.
+Click :guilabel:`Add A Product` and select an item. Click :guilabel:`Add Shipping`, select a
+shipping method, then click :guilabel:`Get Rate`, and finally click :guilabel:`Add`.
```
Change phrasing
```suggestion
shipping method, then click :guilabel:`Get Rate`, and finally, :guilabel:`Add`.
```

hojo-odoo reviewed (commented):
Nice work. Most of my comments are about phrasing or missing dashes. Let me know if you have any questions.

larm-odoo commented:
Thanks for the review, @hojo-odoo ! I implemented all the changes, and it looks like it is ready for @StraubCreative to take a look now. Thank you, everyone, for all your help on this- my first, massive PR!

samueljlieber commented:
Fixed merge conflicts in https://github.com/odoo/documentation/commit/0fd621336d01824cf2ff2758935a88f5fd54630c

samueljlieber commented:
Implemented minor technical changes in https://github.com/odoo/documentation/commit/7c13e8da7e71564e31a265396fa2c4403522b3e6.

- added :ref: links instead of :doc: references 
- minor :guilabel: & :menuselection: corrections
- verified redirects & structural changes

@StraubCreative this PR is ready for your final review 🙂

jcs-odoo commented on redirects/14.0.txt:
```diff
@@ -42,8 +42,18 @@ applications/general/calendars/google/google_calendar_credentials.rst applicatio
 # applications/inventory
 
-applications/inventory_and_mrp/inventory/routes/concepts/inter_warehouse.rst applications/inventory_and_mrp/inventory/management/warehouses/warehouse_replenishment_transfer.rst
+applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst applications/inventory_and_mrp/inventory/shipping/operation/dropshipping.rst                                                  # /inventory/management/incoming/* --> /shipping/operation/*
```
to update here in odoo/odoo in 16.0
https://github.com/odoo/odoo/blob/47333b5b13823ac2812a120f2cb443cb68d7d92c/addons/purchase_stock/views/res_config_settings_views.xml#L20

jcs-odoo replied:
The comment are not correct but it doesn't matter that much ^^ 
- you wrote /inventory/ in all of them, implying that it's removed 
- on this line you wrote incoming instead of delivery

```suggestion
applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst applications/inventory_and_mrp/inventory/shipping/operation/dropshipping.rst                                                  # /management/delivery/* --> /shipping/operation/*
```

jcs-odoo reviewed (commented):
Can you please ping me on the fw-ported branch (either 16.0 or even 16.1) so I can update the dropshipping link in the app's settings, following your redirect rule? And then do not r+ these last PRs immediately so I have the time to create the linked PR.
I don't want to do it for older versions, as there is close to no chance that this update would reach anyone anyway ^^Thanks!

StraubCreative reviewed (changes requested):
Hi all
@larm-odoo @hojo-odoo @brse-odoo @Felicious 

Pretty good lift on these docs!

The biggest revision I have here is to combine the `shipping_deliveries.rst` doc with the `inventory_flow.rst` doc.

The other suggestions below, which may seem voluminous, are really the same formatting suggestions repeated over and over.
- guilabel use
- heading titles
- admonition blocks
- etc.

As far as the other docs `dropshipping.rst` and `label_type.rst` docs go, ideally these would have been submitted as separate [MOV] and [ADD] PRs, respectively. The changes though, seem fine at first glance however there are some things we can follow-up on later (e.g. *Chatter* doesn't get a guilabel, not sure if capitalized either).

I'm going to push up all of the changes in a moment and will merge right away just to ship something and get the changes locked in.

In follow up PRs, I'd like to see:
- update 3-step docs with suggestions I made here
- cleanup `label_type.rst`

Thanks!

StraubCreative commented:
As of [c91b7b3](https://github.com/odoo/documentation/pull/3762/commits/c91b7b377016c7bcdedb586dddf6e5aae3d8d4ed)
- Removed "How to" from h1 heading in `label_type.rst`
- **documentation_guidelines**: missing backtick on `abbr` tag
- **documentation**: bad `doc` path in `stock_warehouses`

All checks pass ✅ 

@robodoo r+
